openbb_platform/extensions/crypto/openbb_crypto/price/price_router.py

```diff
@@ -0,0 +1,27 @@
+# pylint: disable=W0613:unused-argument
+"""Comparison Analysis Router."""
```

Comment:
Highlighting that there is an incorrect label at the top of the router here.

Reply:
Thanks! Fixed.